src/NeutralLandscapes.jl

```diff
@@ -3,56 +3,41 @@ module NeutralLandscapes
 import NaNMath
 using StatsBase: sample
 using Random: rand!
-using Statistics: quantile
-using Distributions: Normal
 using Statistics: quantile, mean
+using Distributions: Normal
 using NearestNeighbors: KDTree, knn, nn
 using DataStructures: IntDisjointSets, union!, find_root, push!
+using Base: @kwdef
 
-"""
-All algorithms are descended from the `NeutralLandscapeMaker` type. A new
-algorithm must minimally implement and `_landscape!` method for this type.
-"""
-abstract type NeutralLandscapeMaker end
-export NeutralLandscapeMaker
-
-include("landscape.jl")
 export rand, rand!
+export classify!, classify, blend, label
 
-include(joinpath("algorithms", "nogradient.jl"))
-export NoGradient
-
-include(joinpath("algorithms", "planargradient.jl"))
-export PlanarGradient
-
-include(joinpath("algorithms", "edgegradient.jl"))
-export EdgeGradient
+export NeutralLandscapeMaker
 
-include(joinpath("algorithms", "diamondsquare.jl"))
+export DiscreteVoronoi
 export DiamondSquare, MidpointDisplacement
```

Comment:
good idea to give mpd it's own file

Reply:
I think it's mostly an application of DiamondSquare, but not opposed to the idea - @gottacatchenall ?